packages/core/src/styles/tokens/tokens.stories.ts

```diff
@@ -192,6 +198,21 @@ function setNumberToken(token: Token) {
   </div>`;
 }
 
+function setAnimationToken(token: Token) {
+  token.demo = html`
+    <div
+      class="animation-demo"
+      style="${token.name.includes('duration') ? '--duration' : '--easing'}: var(${token.name}); ${token.name.includes(
+        'easing'
+      )
+        ? '--duration: var(--cds-global-animation-duration-slower)'
```

Comment:
I'm not sure if it applies here but the CSS custom property convention I've used in the accordion and overlay/modal is `--animation-duration` and `--animation-easing`.